### ISSUE TITLE
Lines between items in the listView

### ISSUE THREAD

**Issue by ouxiaoyong** [flutter/flutter#58462]
**there have an line between items in the listView**

![device-2020-06-02-113707](https://user-images.githubusercontent.com/25840200/83477420-c80c8e00-a4c5-11ea-84f7-86c7a1cb81b5.png)

**### video**
[device-2020-06-02-110230.mp4](https://github.com/ouxiaoyong/IssuesResources/blob/master/device-2020-06-02-113742.mp4)

**flutter doctor -v**

```
[✓] Flutter (Channel stable, v1.17.1, on Mac OS X 10.15.4 19E287, locale zh-Hans-CN)
    • Flutter version 1.17.1 at /Users/domobile/Documents/flutter_sdk/flutter
    • Framework revision f7a6a7906b (3 weeks ago), 2020-05-12 18:39:00 -0700
    • Engine revision 6bc433c6b6
    • Dart version 2.8.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/domobile/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.2

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] IntelliJ IDEA Community Edition (version 2019.2.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (1 available)
    • ONEPLUS A5010 • 636a6231 • android-arm64 • Android 9 (API 28)
```

**### dart code:**

```
Random random;

  @override
  void initState() {
    super.initState();
    random = Random.secure();
  }

  @override
  Widget build(BuildContext context) {
    MediaQueryData mediaQueryData = MediaQuery.of(context);
    print(mediaQueryData);
    return Material(
      color:Colors.transparent,
      child: Container(
        decoration: BoxDecoration(
          color: Colors.red,
        ),
        child: ListView.builder(itemBuilder: _buildItem,
          itemCount: 38,
          padding: EdgeInsets.zero,),
      ),
    );
  }


  Color color;
  Widget _buildItem(_,int index){
    color ??= Color.fromARGB(255, random.nextInt(255), random.nextInt(255), random.nextInt(255));
    return Container(
      height: 300,
      decoration: BoxDecoration(
        color: color,
      ),
      child: Align(
        alignment: Alignment.topCenter,
        child: Material(
          color: Colors.transparent,
          child: Text('item$index',
          style: TextStyle(
            fontSize: 30
          ),),
        ),
      ),
    );
  }
```

**Comment by TahaTesser** [comment]
**Line Is appearing and disappearing on scroll**

![flutter_01](https://user-images.githubusercontent.com/48603081/83516647-1a18d800-a4f5-11ea-83f2-eeb2de1cdadd.png)

![flutter_02](https://user-images.githubusercontent.com/48603081/83516654-1c7b3200-a4f5-11ea-93a3-4e2403c33fbe.png)



<details> 
<summary>Code Sample</summary> 

``` 
import 'dart:math';

import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatefulWidget {
  @override
  _HomeState createState() => _HomeState();
}

class _HomeState extends State<Home> {
  Random random;
  Color color;

  @override
  void initState() {
    super.initState();
    random = Random.secure();
  }

  @override
  Widget build(BuildContext context) {
    MediaQueryData mediaQueryData = MediaQuery.of(context);
    print(mediaQueryData);
    return Scaffold(
      appBar: AppBar(
        title: Text('Material App Bar'),
      ),
      body: Material(
        color: Colors.transparent,
        child: Container(
          decoration: BoxDecoration(
            color: Colors.red,
          ),
          child: ListView.builder(
            itemBuilder: _buildItem,
            itemCount: 38,
            padding: EdgeInsets.zero,
          ),
        ),
      ),
      floatingActionButton: FloatingActionButton(
        child: Icon(Icons.add),
        onPressed: () {},
      ),
    );
  }

  Widget _buildItem(_, int index) {
    color ??= Color.fromARGB(
        255, random.nextInt(255), random.nextInt(255), random.nextInt(255));
    return Container(
      height: 300,
      decoration: BoxDecoration(
        color: color,
      ),
      child: Align(
        alignment: Alignment.topCenter,
        child: Material(
          color: Colors.transparent,
          child: Text(
            'item$index',
            style: TextStyle(fontSize: 30),
          ),
        ),
      ),
    );
  }
}

``` 

</details> 

<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.19.0-2.0.pre, on Mac OS X 10.15.5 19F96, locale
    en-GB)
    • Flutter version 1.19.0-2.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision 1d395c5e18 (2 days ago), 2020-05-31 07:41:50 -0700
    • Engine revision 923687f0e7
    • Dart version 2.9.0 (build 2.9.0-11.0.dev 6489a0c68d)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (3 available)
    • macOS      • macOS      • darwin-x64     • Mac OS X 10.15.5 19F96
    • Web Server • web-server • web-javascript • Flutter Tools
    • Chrome     • chrome     • web-javascript • Google Chrome 83.0.4103.61

• No issues found!
``` 

</details> 

Could be related https://github.com/flutter/flutter/issues/29702

**Comment by HansMuller** [comment]
This looks like a known problem with antialiasing and mapping from logical pixels to device pixels. There's an open issue for a documentation fix: #14288

More info about the problem can be found here:
https://github.com/flutter/flutter/issues/14288#issuecomment-361375791
https://github.com/flutter/flutter/issues/17084#issuecomment-385718108
https://github.com/flutter/flutter/issues/15035#issuecomment-370028740
https://github.com/flutter/flutter/issues/44008

**Comment by goderbauer** [comment]
I am closing this as a known issue per the comment above. Please follow the linked issues.
